rosidl_typesupport_freertps_cpp/resource/msg__type_support.cpp.template

```diff
@@ -12,6 +12,7 @@
 @#  - get_header_filename_from_msg_name (function)
 @#######################################################################
 @
+
```

Comment:
Don't add an empty line here. Look in the generated code - there is already an empty above this Empy comment block.